src/me/goddragon/teaseai/api/texttospeech/TacotronWaveglowProvider.java

```diff
@@ -51,7 +51,7 @@ public void playFetchedAudio(String text, float gainValue, boolean daemon, boole
         try {
             URL url = new URL("http://localhost:8080/tts/get/sound.wav");
             MediaHandler.getHandler().playAudioWithURI(url.toExternalForm(), join);
-        } catch (MalformedURLException | URISyntaxException e) {
+        } catch (MalformedURLException e) {
```

Comment:
This would appear to be required on Windows at least.
```
> Task :compileJava
C:\taj\TeaseAIJava\src\me\goddragon\teaseai\api\config\VariableHandler.java:196: warning: [removal] ScriptObjectMirror in jdk.nashorn.api.scripting has been deprecated and marked for removal
            if (value instanceof ScriptObjectMirror && ((ScriptObjectMirror) value).isArray() || value instanceof Collection) {
                                 ^
C:\taj\TeaseAIJava\src\me\goddragon\teaseai\api\config\VariableHandler.java:196: warning: [removal] ScriptObjectMirror in jdk.nashorn.api.scripting has been deprecated and marked for removal
            if (value instanceof ScriptObjectMirror && ((ScriptObjectMirror) value).isArray() || value instanceof Collection) {
                                                         ^
C:\taj\TeaseAIJava\src\me\goddragon\teaseai\api\config\VariableHandler.java:199: warning: [removal] ScriptObjectMirror in jdk.nashorn.api.scripting has been deprecated and marked for removal
                if (value instanceof ScriptObjectMirror) {
                                     ^
C:\taj\TeaseAIJava\src\me\goddragon\teaseai\api\config\VariableHandler.java:200: warning: [removal] ScriptObjectMirror in jdk.nashorn.api.scripting has been deprecated and marked for removal
                    String[] strings = ((ScriptObjectMirror) value).to(String[].class);
                                         ^
C:\taj\TeaseAIJava\src\me\goddragon\teaseai\api\texttospeech\TacotronWaveglowProvider.java:53: error: unreported exception URISyntaxException; must be caught or declared to be thrown
            MediaHandler.getHandler().playAudioWithURI(url.toExternalForm(), join);
                                                      ^
C:\taj\TeaseAIJava\src\me\goddragon\teaseai\gui\settings\PersonalitySettings.java:115: warning: [removal] ScriptObjectMirror in jdk.nashorn.api.scripting has been deprecated and marked for removal
                if (entry.getValue() != null && entry.getValue().getClass().isArray() || entry.getValue() instanceof ScriptObjectMirror && ((ScriptObjectMirror) entry.getValue()).isArray()) {
                                                                                                                     ^
C:\taj\TeaseAIJava\src\me\goddragon\teaseai\gui\settings\PersonalitySettings.java:115: warning: [removal] ScriptObjectMirror in jdk.nashorn.api.scripting has been deprecated and marked for removal
                if (entry.getValue() != null && entry.getValue().getClass().isArray() || entry.getValue() instanceof ScriptObjectMirror && ((ScriptObjectMirror) entry.getValue()).isArray()) {
                                                                                                                                             ^
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
1 error
6 warnings

> Task :compileJava FAILED

FAILURE: Build failed with an exception.
```

Reply:
Thanks for checking that.  Curiously for me, I can't build a jar file with that exception in.  I'll take it out of the branch.